sockeye/average.py

```diff
@@ -166,7 +169,12 @@ def main():
         help="either a single model directory (automatic checkpoint selection) "
              "or multiple .params files (manual checkpoint selection)")
     params.add_argument(
-        "--max", action="store_true", help="maximize metric (default: min)")
+        "--max", action="store_true", help="Maximize metric.")
+    params.add_argument(
+        "--metric",
+        help="Name of the metric to choose n-best checkpoints from. (default: {})".format(C.PERPLEXITY),
+        default=C.PERPLEXITY,
+        choices=[C.PERPLEXITY, C.BLEU])
```

Comment:
I think we can also add accuarcy here (its not tracked by default anymore but is generally supported)